marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/jobs/job-results-lib.sjs

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
This library is currently based on step responses, which are part of job documents. And the scope is querying for data. "job-results" is a little vague - are these query results, or results from executing a job?

We have "entity-search-lib" already. "search" unfortunately implies use of the Search API.

I am thinking "(noun)-query-lib" is thus the better direction, as "query" doesn't imply a particular API that we're using - which shouldn't matter for the caller either.

So I think this should be "job-query-lib".